Vagrantfile

```diff
@@ -54,16 +54,16 @@ $kube_provider_boxes = {
       # your own value so long as you provide :box_url; for example, the
       # "official" name of this box is "rickard-von-essen/
       # opscode_fedora-20", but by providing the URL and our own name, we
-      # make it appear as yet another provider under the "kube-fedora20"
+      # make it appear as yet another provider under the "kube-fedora22"
       # box
-      :box_name => 'kube-fedora20',
-      :box_url => 'https://atlas.hashicorp.com/rickard-von-essen/boxes/opscode_fedora-20/versions/0.4.0/providers/parallels.box'
+      :box_name => 'kube-fedora22',
+      :box_url => 'https://opscode-vm-bento.s3.amazonaws.com/vagrant/parallels/opscode_fedora-22_chef-provisionerless.box'
```

Comment:
I just saw the comments on lines 54-58 doesn't make sense anymore, should I remove those lines or all (from 49-58)?